terraform/files/bin/delete_cluster.sh

```diff
@@ -90,5 +135,6 @@ if grep '^OLD_OPENSTACK_CLOUD:' $CCCFG >/dev/null 2>&1; then
     echo "Please delete application credential $PREFIX-$CLUSTER_NAME-appcred once capo has cleaned everything up."
   fi
 fi
-# TODO: Clean up ~/$CLUSTER_NAME
+rm -rf ~/$CLUSTER_NAME
+echo "Deleting cluster $CLUSTER_NAME done."
```

Comment:
```suggestion
if test $RC = 0; then
  echo "Deleting cluster $CLUSTER_NAME completed successfully."
else
  echo "Deleting cluster $CLUSTER_NAME likely incomplete."
fi
```

Reply:
Just to avoid reporting that something has been done and then returning a contradicting error code that most likely will be ignored.
This is a rather cosmetic thing though.